docs/workforce-management-framework/upstream-1/master.adoc

```diff
@@ -47,18 +43,23 @@ include::topics/con-datasync-module.adoc[leveloffset=+2]
 
 == Security
 include::topics/con-introducing-securityfeatures.adoc[leveloffset=+2]
+
+== Security {Keycloak}
```

Comment:
[Optional] Not sure about `Security {Keycloak}`, it doesn't sound right to me. I think just `Keycloak` is fine?